prov/efa/src/efa_base_ep.h

```diff
@@ -64,6 +64,44 @@
 #include "ofi_util.h"
 #include "ofi_file.h"
 
+extern struct fi_provider efa_prov;
+
+#define EFA_PROV_NAME "efa"
+#define EFA_WARN(subsys, ...) FI_WARN(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_TRACE(subsys, ...) FI_TRACE(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO(subsys, ...) FI_INFO(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO_ERRNO(subsys, fn, errno) \
+	EFA_INFO(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_WARN_ERRNO(subsys, fn, errno) \
+	EFA_WARN(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_DBG(subsys, ...) FI_DBG(&efa_prov, subsys, __VA_ARGS__)
+
+/*
+ * Specific flags and attributes for shm provider
+ */
+#define EFA_SHM_MAX_AV_COUNT       (256)
+/* maximum name length for shm endpoint */
+#define EFA_SHM_NAME_MAX	   (256)
+
+struct efa_av {
```

Comment:
cannot have efa_av definition in `efa_base_ep.h`. Can you put it in `efa_av.h` and have `efa_base_ep.h` include it?

Reply:
Done